### PR TITLE
Inlining html/jsp code into JSP pages, making less use of the toHTML methods in the POJOs

### PR DESCRIPTION
**Pull request by jadelkhoury**
This patch is not yet tested. Please do not review yet.

### CODE REVIEW

**Comment by jadelkhoury** [comment]
-1 NOT Verified yet.

**Comment by jadelkhoury** [comment]
I have done some testing now. Ready for review. As usual, maybe easier if someone wants to review the produced java code instead of the acceleo templates.

Note: this new jsp code will not be produced unless one deletes the existing jsp pages. This is because we are placing the whole generated jsp code within a single UserProtected code block. So this new code is practically only available for new generations.

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl [hunk 1]
```diff
@@ -68,8 +69,13 @@ To revert to the default generated content, delete all content in this file, and
...
 <%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
 
+<%@page import="org.eclipse.lyo.oslc4j.core.model.Link" %>
 <%@page import="org.eclipse.lyo.oslc4j.core.model.ServiceProvider"%>
-<%@page import="java.util.List" %>
+<%@page import="java.net.URI"%>
+<%@page import="java.util.Date"%>
+<%@page import="java.util.HashSet"%>
+<%@page import="java.util.List"%>
+<%@page import="java.util.Iterator"%>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl]: Do you need all of these types?

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/services/resourcePropertyToHtml.mtl [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    out.write("<em>null</em>");
+}
+else {
+    %>
+    <jsp:include page="[resourceToHtmlJspRelativeFileName(aProperty.range->first(), contextAdaptorInterface)/]">
+        <jsp:param name="asLocalResource" value="true"/>
+        </jsp:include>
+    <%
+}
+%>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/services/resourcePropertyToHtml.mtl]: yeah, this is hacky :)

**Pull request merged**